site/ja/tutorials/keras/basic_classification.ipynb

```diff
@@ -797,7 +796,7 @@
         "    thisplot = plt.bar(range(10), predictions_array, color=\"#777777\")\n",
         "    plt.ylim([0, 1])\n",
         "    predicted_label = np.argmax(predictions_array)\n",
-        "\n",
```

Comment:
I also noticed that this sample code doesn't work if I just copy & paste because of unaligned indent.